dagger/load-test/run-load-test-native.sh

```diff
@@ -0,0 +1,7 @@
+STACK_NAME=dagger-example-native
```

Comment:
If GraalVM native option is not ready yet, do we need this script at the moment?

Reply:
no, will remove